.tsqueryrc.json

```
@@ -0,0 +1,215 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ribru17/ts_query_ls/refs/heads/master/schemas/config.json",
+  "parser_install_directories": ["${HOME}/.local/share/nvim/site/parser"],
+  "parser_aliases": {
+    "ecma": "javascript",
+    "jsx": "javascript"
+  },
+  "valid_captures": {
+    "context": {
+      "context": "specifies that the first line of this node will be used for the context",
+      "context.start": "specifies that the context starts with the start of this node",
+      "context.end": "specifies that the context ends before the start of this node",
+      "context.final": "specifies that the context ends before the end of this node"
+    }
+  },
+  "valid_predicates": {
+    "eq": {
+      "any": true,
+      "parameters": [
+        {
+          "type": "capture",
+          "arity": "required"
+        },
+        {
+          "type": "any",
+          "arity": "required"
+        }
+      ],
+      "description": "checks for equality between two nodes, or a node and a string"
+    },
+    "any-of": {
+      "parameters": [
+        {
+          "type": "capture",
+          "arity": "required"
+        },
+        {
+          "type": "string",
+          "arity": "required"
+        },
```

```suggestion
        },
        {
          "type": "string",
          "arity": "required"
        },
```

this mandates any-of take 2+ strings